src/components/chartArea/chartArea.js

```diff
@@ -124,18 +124,22 @@ class ChartArea extends React.Component {
 
   render() {
     const { chartWidth } = this.state;
-    const { dataSetOne } = this.props;
+    const { dataSetOne, xAxisFixLabelOverlap, yAxisFixLabelOverlap } = this.props;
 
     const { xAxisTickValues, xAxisTickFormat, yAxisTickValues, yAxisTickFormat } = this.getChartTicks();
-    const updatedXAxisProps = {};
+    const updatedXAxisProps = {
+      fixLabelOverlap: xAxisFixLabelOverlap
+    };
     const updatedYAxisProps = {
       dependentAxis: true,
-      showGrid: true
+      showGrid: true,
+      fixLabelOverlap: yAxisFixLabelOverlap
     };
 
+    console.log('xAxisFixLabelOverlap - - -- - ->', xAxisFixLabelOverlap);
```

Comment:
console.log 😄

Reply:
yup, looks like i caught it at the same time